config/config.go

```diff
@@ -69,6 +82,108 @@ type Config struct {
 	RecoverPolicy            string        `toml:"recover_policy"`
 }
 
+type SnapshotterConfig struct {
+	StartupFlag command.Args `toml:"snapshotter"`
+}
+
+func LoadShotterConfigFile(snapshotterConfigPath string, config *SnapshotterConfig) error {
```

Comment:
LoadSnapshotterConfigFile ?

Reply:
Can we return snapshotter's config from the function rather using output parameters?